plugins/doc_fragments/vultr_v2.py

```diff
@@ -38,6 +38,14 @@ class ModuleDocFragment(object):
       - Fallback environment variable C(VULTR_API_RETRY_MAX_DELAY).
     type: int
     default: 12
+  api_results_per_page:
+    description:
+      - When receiving large numbers of resources, specify how many results should be returned per call to API.
+      - This does not determine how many results are returned; all resources are returned according to other filters.
+      - Vultr API maximum is 500.
+      - Fallback environment variable C(VULTR_API_RESULTS_PER_PAGE).
+    type: int
+    default: 100
```

Comment:
we want this in the next release, so you may add the version, when it is going to be added

```diff
  default: 100
+ version_added: 1.14.0
```